src/CQRS/LeanCode.CQRS.AspNetCore/Registration/CQRSApiDescriptionConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections.Frozen;
+using Microsoft.AspNetCore.Routing;
+
+namespace LeanCode.CQRS.AspNetCore.Registration;
+
+public class CQRSApiDescriptionConfiguration
+{
+    // Required by Swagger (Swashbuckle)
+    public static readonly FrozenDictionary<string, string?> DefaultRouteValues = new Dictionary<string, string?>
+    {
+        ["controller"] = "CQRS"
+    }.ToFrozenDictionary();
+
+    public Func<RouteEndpoint, IDictionary<string, string?>> RouteValuesMapping { get; init; } =
```

Comment:
Will IReadOnlyDictionary work?

Reply:
AFAIK no, the `RouteValues` of `ActionDescriptor` are of the `IDictionary` type.